Packs/Confluera/Widgets/widget-IqHubLog.yml

```diff
@@ -0,0 +1,60 @@
+commonfields:
```

Comment:
As this is an automation script, this file should be in a directory under the `Scripts` directory, and not in the `Widgets` directory, and the filename should not start with `widget-`